package.json

```diff
@@ -158,6 +158,7 @@
     "dotenv": "^8.2.0",
     "dotenv-expand": "^5.1.0",
     "enzyme": "^3.11.0",
+    "eslint": "^7.32.0",
```

Comment:
if adding eslint as part of this PR shouldn't we add the .eslint config? Or is that post prettier?

Reply:
my mistake - added in.